SONAR-netCDF4/docs/tableAttitude_sub_group.adoc

```diff
@@ -0,0 +1,54 @@
+|==============================================================================================================================================================================================================
+|Description |Obligation |Comment
+|Group attributes | |
+|:string |O |System description
+|Dimensions | |
+|time | |can be fixed or unlimited length, as appropriate
+|Coordinate variables | |
+|uint64 time(time) | |Time from attitude sensor
+|:axis = "T" | |
+|:calendar = "gregorian" | |
+|:long_name = "Timestamps for attitude data" | |
+|:standard_name = "time" | |
+|:units = "nanoseconds since 1601-01-01 00:00:00Z" | |
+|Variables | |
+|float distance(time) |O |Distance travelled by the platform from an arbitrary location.
```

Comment:
Is this part of the attitude instrument? It typically comes from a doppler log or the GPS.

Reply:
I move it to the position group. If it comes from another sensor, it might be added as a new subgroup